chirp/data/pipeline.py

```diff
@@ -866,7 +866,17 @@ def __call__(
         for name, feature in features.items()
         if name in self.names
     }
+  
+@dataclasses.dataclass
+class FilterMultiLabelRecordings(DatasetPreprocessOp):
+  """Filters out recordings that have multiple foreground labels."""
 
+  def __call__(
+      self, dataset: tf.data.Dataset, dataset_info: tfds.core.DatasetInfo
+  ) -> tf.data.Dataset:
+    def _predicate(features):
+      return len(features['label']) == 1
```

Comment:
I'm not sure this would work, since we are in graph-mode when calling `__call__`. Use `tf.math.equal(tf.shape(features['label'])[0], 1)` to be sure.